### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'profiles/defaults.json'

### ISSUE THREAD

**Issue by koenvervloesem** [rhasspy/rhasspy#39]
I installed rhasspy-voltron in a virtual environment, following the [installation instructions](https://rhasspy.github.io/rhasspy-voltron/installation.html), then I entered the virtual environment and ran `rhasspy-voltron`:

```
source .venv/bin/activate
./bin/rhasspy-voltron --profile en
```
This fails with:

```
Starting up...
DEBUG:__main__:Namespace(debug=True, docker_compose='', local_mqtt_port=12183, profile='en', supervisord_conf='supervisord.conf', system_profiles=PosixPath('profiles'), user_profiles=PosixPath('/home/pi/.config/rhasspy/profiles'))
DEBUG:__main__:Loading profile en
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/pi/rhasspy-voltron/rhasspy-supervisor/rhasspysupervisor/__main__.py", line 91, in <module>
    main()
  File "/home/pi/rhasspy-voltron/rhasspy-supervisor/rhasspysupervisor/__main__.py", line 69, in main
    profile = Profile(args.profile, args.system_profiles, args.user_profiles)
  File "/home/pi/rhasspy-voltron/rhasspy-profile/rhasspyprofile/__init__.py", line 46, in __init__
    self.load_profile()
  File "/home/pi/rhasspy-voltron/rhasspy-profile/rhasspyprofile/__init__.py", line 82, in load_profile
    with open(defaults_path, "r") as defaults_file:
FileNotFoundError: [Errno 2] No such file or directory: 'profiles/defaults.json'
```

Also reported [on the forum](https://community.rhasspy.org/t/rhasspy-voltron-startup-fails-no-such-file-or-directory-profiles-defaults-json/951).

**Comment by deorder** [comment]
It cannot find the core profile. Make sure the rhasspy-profile submodule is on the master branch.

I just updated everything to the master branch myself:

```
git submodule foreach git pull origin master
git submodule foreach git checkout master
```

**Comment by koenvervloesem** [comment]
Thanks for the pointer, I made it work by updating everything to the master branch like you suggested and doing a `pip install supervisor` in the venv and a `sudo apt install mosquitto`.

**Comment by NullEnt1ty** [comment]
It seems that the submodule "rhasspy-profile" was "upgraded" to a much older revision (probably by accident).

It was before [@ a174cb0](https://github.com/rhasspy/rhasspy-profile/tree/a174cb0043ebfebea5e9c66b6ba1097cfcffcf7b) and is now [@ fc3cabd](https://github.com/rhasspy/rhasspy-profile/tree/f6cceba70a24cc39a75138811373fc501e5935f0).

[This was the commit](https://github.com/rhasspy/rhasspy-voltron/commit/b7d745a40d2445c8b015547feacdf8fe196ea1ce) that introduced this change. @maxbachmann was this change intentionally? 😄 

**Edit**:

Same story with "rhasspy-supervisor".

**Comment by maxbachmann** [comment]
Ouch ... no this was definetly an accident (No clue how I managed to do this :sweat_smile:). All submodules should be on the latest version now again. Thanks for finding this @NullEnt1ty

**Comment by NullEnt1ty** [comment]
@maxbachmann I can confirm that it works for me now. Thanks!

**Comment by koenvervloesem** [comment]
Fixed by 3f40f54c0600f2ce02912deeee137aa0fd32df64
